### PR TITLE
(GH-1088) Fixed typo/spelling

### PR DESCRIPTION
**Pull request by dragon788**
Cooling the burn.

### CODE REVIEW

File: src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1 [hunk 1]
```diff
@@ -68,66 +68,66 @@ this parameter.
...
+values are 'md5', 'sha1', 'sha256' or 'sha512' - defaults to
+ChecksumType parameter value.
+
+MD5 is not recommended as certain organizations need to use FIPS
+compliant algorithms for hashing - see
 https://support.microsoft.com/en-us/kb/811833 for more details.
-
-The recommendation is to use at least SHA256.
+
+The recommendation is to use at least SHA256.
```
Comment (ferventcoder) [inline src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1]: What happened to the rest of the file?

File: src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1 [hunk 2]
```diff
@@ -51,7 +51,7 @@ it in the package next to the install script, the path will be like
 This is the 32 bit url to download the resource from. This resource can
 be used on 64 bit systems when a package has both a Url and Url64bit
 specified if a user passes `--forceX86`. If there is only a 64 bit url
-available, please remove do not use the paramter (only use Url64bit).
+available, please remove do not use the parameter (only use Url64bit).
```
Comment (ferventcoder) [inline src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1]: +1 - you don't need a ticket for a typo fix, especially if it is in docs. Just send a PR with `(doc) Fixed typo/spelling`. HTH.

File: src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1 (hunk 1 above)
Comment (dragon788) [inline src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1]: I'm betting it was Vim and/or Git messing with the line endings, nothing seems to be missing just "changed".

**Comment by ferventcoder** [comment]
Merged into stable at https://github.com/chocolatey/choco/commit/50621cd41f4e299dd6d65ffd62da245fb674ef8b

**Pull request closed without merging**